helix-core/src/main/java/org/apache/helix/spectator/RoutingDataCache.java

```diff
@@ -84,13 +104,23 @@ public synchronized void refresh(HelixDataAccessor accessor) {
       LOG.info("Reload CurrentStates. Takes " + (System.currentTimeMillis() - start) + " ms");
     }
 
+    if (_sourceDataType.equals(PropertyType.CUSTOMIZEDVIEW) && _propertyDataChangedMap
+        .get(HelixConstants.ChangeType.CUSTOMIZED_VIEW) && _aggregationType != null) {
```

Comment:
If _aggregationType is empty, do we need to to the following refresh?

Reply:
Fixed. Thanks.